vendor/text-encoding/index.js

```diff
@@ -0,0 +1,9 @@
+// This is free and unencumbered software released into the public domain.
+// See LICENSE.md for more information.
+
+var encoding = require("./lib/encoding.js");
```

Comment:
Where will `encoding.js` be loaded from?  Should that be included in the bundle?

Reply:
It's loaded from the path as the vendor directory is part of the published module. They are not part of the bson.rn.cjs artefact, hence it adds the following lines:

```
const { TextEncoder, TextDecoder } = require('../vendor/text-encoding');
const { encode: btoa, decode: atob } = require('../vendor/base64');
```

Reply:
But we need to include an `encoding.js` file somewhere, right?  sorry for these dumb questions:  

Where is this file?  Is it generated?  I don't see it in our existing bson source, or generated `lib`, or in this PR, and neither library was added as a dependency so it won't come from there.

also, failing tests: https://spruce.mongodb.com/task/node_bson_linux_no_bigint_web_tests_patch_259547d9e926c87f61f664a68c2a3a9172f53d2c_647790670305b946d4e34eee_23_05_31_18_22_32/logs?execution=0

Reply:
I see it's included in the release test - it might be an accident that it didn't make it into the PR?

Reply:
Yeah I seemed to have missed it in the PR.